heka/sandbox/decoders/extract_telemetry_dimensions.lua

```diff
@@ -102,6 +102,7 @@ function process_message()
         msg.Fields.appBuildId       = app.buildId
         msg.Fields.vendor           = app.vendor
         msg.Fields.clientId         = parsed.clientId
+        msg.Fields.creationDate     = parsed.creationDate
 
```

Comment:
If our aim is to be able to use it for calculations, then we might as well parse it - if you want the "raw" string version you can always parse it out of the payload...  I agree with avoiding surprises downstream, but having all the timestamps in the same format is unsurprising in other ways :)